dev-python/ipcalc/ipcalc-1.99.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="IP subnet calculator"
+HOMEPAGE="https://pypi.python.org/pypi/ipcalc/"
+SRC_URI="https://github.com/tehmaze/${PN}/archive/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="dev-python/six[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}-${P}"
+
+python_test() {
+	"${EPYTHON}" ipcalc.py || die "Testing failed under ${EPYTHON}"
```

Comment:
Those are not really tests, they just print some stuff. You're looking for `test.py`… which apparently isn't part of the two-year old 1.99.0 release. Could you look into packaging a snapshot from git master instead, or asking upstream to make a new release?

Reply:
Could be sufficient add on SRC_URI also for test.py from snapshot and I use it ? If we need wait for a new release I think that users wait a lot of time before use a more fresh release.

Reply:
As far as I understand @mgorny's, this file doesn't really test the software. If there's no test, please remove the `python_test()` as well as the `IUSE=test` parts altogheter.

Reply:
Hi, sorry for delay. Few free time in the last weeks. I agreed with test use flag. My fault, I will fix it.

While about test I think that is correct. Currently ipcalc library is very easy, contains a single file ipcalc.py that contains Class IP and Netwask and if it execute directly elaborate a stupid test suite that try to use IP and Netmask classes. If you execute ipcalc on test doesn't test import from external packages but test that library it works. I think that this is correct. WDYT ?

Reply:
Just remove the tests for now.